tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-viewer.html

```diff
@@ -39,7 +39,9 @@
       .header {
         color: #80868B;
         font-size: 12px;
-        padding: 5px 0;
+        padding: 4px 0;
+        border-bottom: solid 1px #DADCE0;
+        border-top: solid 1px #DADCE0;
```

Comment:
is this from mahima feedback?

Reply:
Added a border around table headers in modules to distinguish them from any icons/buttons around them.